### ISSUE TITLE
wombat.js rewrite_html strips td tags

### ISSUE THREAD

**Issue by jcushman** [webrecorder/pywb#138]
There's some funky formatting in the menu for this playback:

![image](https://cloud.githubusercontent.com/assets/376272/10590173/505dbbde-767d-11e5-9ea1-e13b342474e6.png)

(Orig url http://www.gov.cn/gongbao/content/2001/content_61187.htm)

The reason seems to be all the document.writes in http://www.gov.cn/js/utf8/gongbaodaohang_xl.js

In particular, `rewrite_html` messes up the table layout by stripping td tags:

```
> rewrite_html("    <td width=\"110\" height=\"30\"><img src=\"http:\/\/www.gov.cn\/ziliao\/gbgg\/images\/gwygb_5_1.gif\" width=\"110\" height=\"30\"><\/td>", true)
"<img src="http://perma-archives.dev:8000/warc/RVH8-J75M/20151019183556/http://www.gov.cn/ziliao/gbgg/images/gwygb_5_1.gif" width="110" height="30">"
```

I'm not totally clear what rewrite_html is doing, so not sure of the best fix ...

**Comment by ikreymer** [comment]
Thanks for catching this, it is indeed happening:
```
new DOMParser().parseFromString('<td><img src="/foo"/></td>', 'text/html').documentElement.outerHTML
"<html><head></head><body><img src="/foo"></body></html>"
```

Looks like may need to wrap `<td>` in a `<table>` tag, then remove it...

Some more info on this can be found here:
http://krasimirtsonev.com/blog/article/Revealing-the-magic-how-to-properly-convert-HTML-string-to-a-DOM-element

**Comment by ikreymer** [comment]
Another example of issue:

http://www.rrk.ir/ShowPage.aspx?PageCode=40

**Comment by ikreymer** [comment]
Should be fixed in pywb 2.0
